modular_citadel/code/modules/client/loadout/suit.dm

```diff
@@ -77,45 +82,43 @@
 	category = SLOT_WEAR_SUIT
 	path = /obj/item/clothing/suit/hooded/wintercoat/medical
 	restricted_roles = list("Chief Medical Officer", "Medical Doctor") // Reserve it to Medical Doctors and their boss, the Chief Medical Officer
-	
-/* Commented out until there is a Chemistry Winter Coat
-/datum/gear/coat/med/chem
-	name = "Chemistry winter coat"
+
+/datum/gear/coat/robotics
```

Comment:
You should probably say if you add another coat to the loadout, 

and since there is now a Chemistry Winter Coat, you can simply uncomment the section of the chem winter coat while changing the path for "/obj/item/clothing/suit/hooded/wintercoat/chemistry" instead of removing it.

Reply:
>Adds robotics winter coat loudout option
Didn't I? Or should I say it somewhere else? Also, if you look, I uncommented the chem winter coat, not removed it.

Reply:
Thanks, also I think it is Github being stupid, I remember one time someone made a change to a line, and it considered as if it was completely removed and replaced with another line, I think it was the same that happened with the uncommenting.